src/DwitTech.NotificationService.Core/Services/EmailService.cs

```diff
@@ -1,9 +1,103 @@
-﻿using DwitTech.NotificationService.Core.Interfaces;
+﻿using DwitTech.NotificationService.Core.Dtos;
+using DwitTech.NotificationService.Core.Interfaces;
+using DwitTech.NotificationService.Data.Context;
+using DwitTech.NotificationService.Data.Entities;
+using DwitTech.NotificationService.Data.Repository;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Gmail.v1;
+using Google.Apis.Gmail.v1.Data;
+using Google.Apis.Services;
+using Google.Apis.Util.Store;
 using Microsoft.Extensions.Configuration;
+using NLog;
+using System.Net;
+using System.Net.Mail;
+
 namespace DwitTech.NotificationService.Core.Services
 {
     public class EmailService : IEmailService
     {
-        
+        private readonly IConfiguration _config;
+        private static Logger logger = LogManager.GetCurrentClassLogger();
```

Comment:
No. instantiate the LogManager in program and inject the ILogger in the constructor here